src/app/participant/rating_calculator.py

```diff
@@ -17,11 +18,18 @@ def process(self):
             participant_score = 0.0
         opponent_score = 1.0 - participant_score
 
-        # temporary; will be replaced with participant's K value later
-        k = 32.0
+        self.participant.rating += self.participant.k_factor * (participant_score - expected_score_participant)
+        self.opponent.rating += self.opponent.k_factor * (opponent_score-expected_score_opponent)
 
-        self.participant.rating += k * (participant_score - expected_score_participant)
-        self.opponent.rating += k * (opponent_score-expected_score_opponent)
+        self.participant.games_played += 1
+        self.opponent.games_played += 1
+
+        # If participant has played more games than the scaling factor,
+        # reduce it to the league's min k_factor
```

Comment:
the scaling factor should be something like how many games it takes until a particpant's k factor reaches its minimum value, instead of instantly happening at some threshold.
so if the max is 32, min is 16, and the scale factor is 100 games, then it takes 100 games for 32 to decay to 16. Or each game they play reduces the participants k factor by (max-min)/(scale factor) or 16/100 in that case